.gitmodules

```
@@ -46,6 +46,10 @@
 	path = perses-operator
 	url = https://github.com/perses/perses-operator
 	branch = release-coo-1.1
+[submodule "perses"]
+	path = perses
+	url = https://github.com/perses/perses.git
+	branch = release-coo-1.1
```

We should use the upstream release branch
```suggestion
	branch = release-v0.50
```